applications/callflow/src/module/cf_ring_group.erl

```diff
@@ -134,6 +134,11 @@ start_builder(EndpointId, Member, Call) ->
       end
      ).
 
+-spec member_active(kz_json:object()) -> boolean().
+member_active(Member) ->
```

Comment:
Generally functions that return booleans are prefixed with `is_` or `should_`. In this case, `is_member_active/1`.